src/editor/zhuyin_layout/pinyin.rs

```diff
@@ -12,16 +12,19 @@ const MAX_PINYIN_LEN: usize = 10;
 /// TODO: docs
 #[derive(Debug, Default, Clone, Copy, PartialEq, Eq)]
 pub enum PinyinVariant {
-    /// TODO: docs
+    /// [Hanyu Pinyin](https://en.wikipedia.org/wiki/Pinyin), or simply pinyin
     #[default]
     HanyuPinyin,
-    /// TODO: docs
+    /// [THL, 台灣華語羅馬拼音](https://pinyin.thl.tw/)
```

Comment:
Don't use Chinese characters in comments unless you have to.

Reply:
/// [THL, taíwan huáyǔ luómǎ pinyin](https://pinyin.thl.tw/)

Reply:
It looks better with title case `Taíwan Huáyǔ Luómǎ Pinyin`

Reply:
updated